src/commands/pull-requests.ts

```diff
@@ -45,7 +45,7 @@ abstract class PullRequestCommand extends TokenCommand {
       } else {
         vscode.window.showWarningMessage(
           `Cannot create pull request without remote branch. ` +
-            `Please push your local branch before creating pull request.`
+          `Please push your local branch before creating pull request.`
```

Comment:
Let me know if you'd like it reverted. VSC did that automatically. :/

Reply:
That's totally fine